bin/ch/WScriptJsrt.cpp

```diff
@@ -37,6 +38,8 @@
 
 MessageQueue* WScriptJsrt::messageQueue = nullptr;
 std::map<std::string, JsModuleRecord>  WScriptJsrt::moduleRecordMap;
+std::map<JsModuleRecord, std::string>  WScriptJsrt::moduleDirMap;
```

Comment:
Filed a separate test issues to drive the necessary changes in ch.exe's global states. Merging this PR to unblock test262 updates.

Issues:
[https://github.com/Microsoft/ChakraCore/issues/3319](https://github.com/Microsoft/ChakraCore/issues/3319)
[https://github.com/Microsoft/ChakraCore/issues/3318](https://github.com/Microsoft/ChakraCore/issues/3318)


---
In reply to: [126265384](https://github.com/Microsoft/ChakraCore/pull/3284#discussion_r126265384) [](ancestors = 126265384,125369988)